src/main/java/dota/buff/model/User.java

```diff
@@ -0,0 +1,18 @@
+package dota.buff.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class User {
+    private String login;
+    private String password;
+    private Key key;
+
+    public void setKey(String apiKey) {
+        this.key.setApiKey(apiKey);
+    }
```

Comment:
seeing smth weird, you have an annotation @Setter, why do you need setKey here?

Reply:
Oh okay, I got your idea, but if u wanna set apiKey like a string to the User then to the Key, it's not a good approach in my view, better to pre-create a Key somewhere and put it within either using the constructor or setKey method.

Reply:
@spikyzero - so I still seeing this setter, which should be removed from here, because @Data will do it for us -> setKey(Key key) {}

Reply:
@spikyzero - so I still seeing this setter, which should be removed from here, because @DaTa will do it for us -> setKey(Key key) {} and we don't need explicitly set apiKey like a string.